README.md

```diff
@@ -18,7 +18,7 @@ CloudFront, for example.
 
 ### Building
 
-[![Build Status](https://travis-ci.com/mmcgrana/gobyexample.svg "Travis CI status")](https://travis-ci.com/mmcgrana/gobyexample)
+[![Build Status](https://github.com/mmcgrana/gobyexample/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/mmcgrana/gobyexample/actions)
```

Comment:
Did you sync your branch?

Reply:
I guess I did. Should I revert this?

Reply:
These changes don't appear in `README.md` any more. Could you please rebase your branch onto `master` (or merge `master` into it) and resend the PR?